targets/drive/mobile/config.xml

```diff
@@ -101,8 +101,10 @@
     </plugin>
     <plugin name="cordova-plugin-x-socialsharing" spec="~5.4.1" />
     <plugin name="cordova-plugin-safariviewcontroller" spec="~1.5.4" />
-    <plugin name="cordova-ios-plugin-no-export-compliance" spec="git+https://github.com/Cordobo/cordova-plugin-ios-no-export-compliance.git" />
-    <plugin name="io.cozy.jsbackgroundservice" spec="https://github.com/cozy/cordova-jsbackgroundservice.git#1.3.0" />
-    <plugin name="io.cozy.plugins.listlibraryitems" spec="https://github.com/cozy/cordova-plugin-list-library-items.git#1.5.0" />
+    <plugin name="cordova-ios-plugin-no-export-compliance" spec="git+https://github.com/Cordobo/cordova-plugin-ios-no-export-compliance.git" />    <plugin name="io.cozy.jsbackgroundservice" spec="https://github.com/cozy/cordova-jsbackgroundservice.git#1.3.0" />
     <plugin name="com-darryncampbell-cordova-plugin-intent" spec="^1.1.1" />
+    <plugin name="cordova-plugin-browsersync" spec="~0.1.7" />
```

Comment:
I don't think this is supposed to be part of this PR?

Reply:
So painful that Cordova 7 reads the package.json to update this one